apps/web/playwright/manage-booking-questions.e2e.ts

```diff
@@ -52,8 +52,21 @@ test.describe("Manage Booking Questions", () => {
       // Considering there are many steps in it, it would need more than default test timeout
       test.setTimeout(testInfo.timeout * 3);
       const user = await createAndLoginUserWithEventTypes({ users });
+      const team = await prisma.team.findFirst({
+        where: {
+          members: {
+            some: {
+              userId: user.id,
+            },
+          },
+        },
+        include: {
+          eventTypes: true,
+        },
```

Comment:
```suggestion
        select: {
          id: true,
          eventTypes: true,
        },
```
We can use this and then below use
const teamId = team?.id ?? 0;